security/src/main/java/io/micronaut/security/authentication/AuthenticationExceptionHandler.java

```diff
@@ -40,15 +44,41 @@
 public class AuthenticationExceptionHandler implements ExceptionHandler<AuthenticationException, MutableHttpResponse<?>> {
     protected final ApplicationEventPublisher eventPublisher;
 
+    private final ErrorResponseProcessor<?> responseProcessor;
+
+    /**
+     * @param eventPublisher The event publisher
+     */
```

Comment:
Add `@deprecated` in javadoc pointing to alternative

Reply:
Done